modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Coverage/CoverageInfoForm.tsx

```diff
@@ -0,0 +1,254 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import { ClaySelect } from '@clayui/form';
+import classNames from 'classnames';
+import { bodilyInjuryOptions, collisionOptions, comprehensiveOptions, medicalOptions, propertyDamageOptions, underinsuredInjuredOptions, uninsuredDamageOptions } from './SelectOptions';
+
+type hasRequiredErrorTypes = {
+  bodilyInjury: boolean,
+  propertyDamage: boolean,
+  underinsuredInjured: boolean,
+  uninsuredDamage: boolean,
+  medical: boolean,
+  comprehensive: boolean,
+  collision: boolean
+};
+
+type HasValidationsTypes = {
+  bodilyInjury: boolean,
+  propertyDamage: boolean,
+  underinsuredInjured: boolean,
+  uninsuredDamage: boolean,
+  medical: boolean,
+  comprehensive: boolean,
+  collision: boolean
+}
+
+const hasRequiredError: hasRequiredErrorTypes = {
+  bodilyInjury: false,
+  propertyDamage: false,
+  underinsuredInjured: false,
+  uninsuredDamage: false,
+  medical: false,
+  comprehensive: false,
+  collision: false
+};
+
+
+const hasValidations: HasValidationsTypes = {
+  bodilyInjury: false,
+  propertyDamage: false,
+  underinsuredInjured: false,
+  uninsuredDamage: false,
+  medical: false,
+  comprehensive: false,
+  collision: false
+}
```

Comment:
Object keys should be sorted in natural ascending order.

Reply:
I will remove it